ThunderTable/DatePickerCell.swift

```diff
@@ -0,0 +1,36 @@
+//
+//  DatePickerCell.swift
+//  ThunderTable
+//
+//  Created by Simon Mitchell on 16/09/2020.
+//  Copyright © 2020 3SidedCube. All rights reserved.
+//
+
+import UIKit
+
+/// A simple protocol that the two variants of date picker `UITableViewCell` subclasses
+/// can conform to so they can share the same `Row` type whilst still providing unique UI
+@objc public protocol DatePickerCell {
```

Comment:
It does feel a little bit more like a superclass rather than protocol, a looooot like in Blood the `DateTableViewCell`.
`DatePickerCell` as a name does sound a lot like a subclass of `UITableViewCell` or `UICollectionViewCell`.
Some might argue the super/sub class OOO is less of a thing in Swift and usually opt for protocol though. Let me know your thoughts

Reply:
That might be a very good shout! Only thing is whether you know if it's possible to `set` an `IBOutlet` (or override a standard var as `IBOutlet`) because on one class `datePicker` needs to be an `IBOutlet` and on the other it doesn't :)

Reply:
Don't think the fact it's an `IBOutlet` matters, that's just for Xcode to connect up the nib and the file. You can use a get property for that :). Can make it "abtract" if you want but that's not really a thing in Swift

Reply:
I had a feeling that was the case, just not something I've ever tried before :P I think subclass is a good shout in that case for now whilst we have just two cells :D

Reply:
I had no idea you can make it "abstract", is that something from objective-c?

Reply:
Abstract is a thing in other programming languages (like Java) where subclasses MUST do something. It's not a thing is Swift and generally avoided. In this case though subclasses might want to inherit a load of properties. Generally protocol is the way to go in Swift. Tbf even apple made the `View` a protocol in SwiftUI.